src/Component/composer.json

```diff
@@ -31,6 +31,7 @@
         "php": "^7.4 || ^8.0",
         "sylius-labs/polyfill-symfony-event-dispatcher": "^1.0",
         "sylius/registry": "^1.5",
+        "symfony/config": "^4.4 || ^5.2",
```

Comment:
Perhaps all dependencies should be bumped to 5.3 according to github workflow, or we should increase amount of builds and test 4.4, 5.2 and 5.3

Reply:
@lchrusciel yes and add a "^6.0" but in a separated PR, what do you think?

Reply:
With Symfony 6.0 is a great idea to start testing. But in terms of this PR I see only two possibilities:
1) Bump all Symfony requirements to 5.3
2) Run pipelines for Symfony 4.4, 5.2, and 5.3
both of them are fine for me, but I would say that 1 is probably simpler so that would be my choice

Reply:
@lchrusciel first choice done